.eslintrc.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
     Headers: false,
     Request: false,
     XMLHttpRequest: false,
+    globalThis: false,
```

Comment:
Wondering if we should/could add a linter rule disallowing the use of `global`?
(I did not find a rule for this after a quick search)

Reply:
I wonder if setting global: true here would work.

Reply:
Will try soon

Reply:
Setting them to "off" works (see https://eslint.org/docs/latest/use/configure/language-options#using-configuration-files-1)